#FN-42 feat: Seller create/add items

Lydie260 opened the pull request:
# PR Description
As a Seller
I should be able to add/create items for sale
This page should only be accessible to sellers, i.e authenticated
Such that the stock can be seen by any requesting client
Handle all responses and errors from the backend.

Please include a brief description of the pull request you want to raise.

## Description of tasks that were expected to be completed

List all the tasks and sub-tasks that were assigned to you Make sure you check every completed task. list them in this style

- [x] Add product to the seller's collection

## Description of tasks that were expected to be completed and that are not yet completed

- [x] Add new product
- [x] Display new added product

## How this may be tested?
Please describe the tests that you ran to verify your functionalities. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration.


0. clone the project on https://github.com/atlp-rwanda/ecommerce-app-cogito-fe.git
1. run `yarn i` to install all required dependencies
2. run `yarn start`
3. Go to the login page for authentication or create a new account if you don't have one N.B login as a seller.
4. After logging as a seller you will be directed to seller's page. 
5. After accessing the seller's page click on add new product fill out the form then click add.

## Screenshots (If appropriate/applicable)

Provide screenshots related to your PR in case it's appropriate and applicable

## Please check this Checklist before you submit your PR:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my code
- [x] I have commented on my code, particularly in hard-to-understand areas
- [x] My code generates no warnings
- [x] There are no vulnerabilities

Honore123 reviewed (commented):
Allow vendors to add at least 4 images per product

Illustre13 committed:
feat(login): worked on login with google (#23)

- connected the google login functionalities of the backend to our frontend
- wrote test case for this PR with jest
[Finishes #45]

dushimeemma reviewed (changes requested):
- Why are you using `npm` while others are using `yarn`
- Enable to test your PR, what are the steps to follow?

Lydie260 commented:
> * Why are you using `npm` while others are using `yarn`
> * Enable to test your PR, what are the steps to follow?
Done with correction

dushimeemma commented on src/App.tsx:
```diff
@@ -6,18 +6,23 @@ import SignupPage from './components/register';
       <div>
         <Router>
-          <NavBar />
+          {/* <NavBar /> */}
```
Why did you comment on this line of code?

Lydie260 replied:
I did it because for dashboard it looks weird to have dashboard and footer

dushimeemma commented on src/App.tsx:
```diff
@@ -28,10 +33,13 @@ const App: React.FC = () => {
+            <Route path="/sellerPage" element={<VendorPage />} />
           </Routes>
-          <Footer />
+          {/* <Footer /> */}
```
Why did you comment this line of code?

Lydie260 replied:
I did it because for dashboard it looks weird to have dashboard and footer
